src/app/modules/contact/contact.component.html

```diff
@@ -1 +1,73 @@
+            <!--                        matTooltipPosition="below"-->
+            <!--                    />-->
+            <!--                </a>-->
+            <!--            </div>-->
+            <mat-card-title class="font-semibold">You Could Also Send Me A Message!</mat-card-title>
+            <!--            <mat-card-subtitle class="dark:text-custom-yellow-500"> You Need Anything? Please Don't Hesitate To Ask! </mat-card-subtitle>-->
+        </mat-card-header>
+
+        <mat-divider inset></mat-divider>
+
+        <mat-card-content>
+            <div class="flex flex-col justify-around">
+                <div class="flex items-center my-5">
+                    <input
+                        id="name"
+                        name="name"
+                        class="order-last input-textarea dark:input-text-area-dark-theme input-text-area-light-theme"
+                        type="text"
+                        placeholder="Name"
+                        required
+                    />
+                    <label for="name" class="input-icon dark:input-icon-dark-theme input-icon-light-theme">
+                        <mat-icon class="order-first">person_outline</mat-icon>
+                    </label>
+                </div>
+
+                <div class="flex items-center my-5">
+                    <input
+                        id="email"
+                        name="email"
+                        class="order-last input-textarea dark:input-text-area-dark-theme input-text-area-light-theme"
+                        type="email"
+                        placeholder="Email"
+                        required
+                    />
+                    <label for="email" class="input-icon dark:input-icon-dark-theme input-icon-light-theme">
+                        <mat-icon class="order-first">email</mat-icon>
+                    </label>
+                </div>
+
+                <div class="flex items-center my-5">
+                    <textarea
+                        id="message"
+                        name="email"
+                        class="order-last px-2 py-5 input-textarea dark:input-text-area-dark-theme input-text-area-light-theme"
+                        type="text"
+                        placeholder="Message"
+                        required
+                    ></textarea>
+
+                    <label for="message" class="textarea-icon dark:input-icon-dark-theme input-icon-light-theme">
+                        <mat-icon class="order-first">email</mat-icon>
+                    </label>
+                </div>
+            </div>
+        </mat-card-content>
+        <mat-card-actions class="mt-3 mx-1 flex justify-end">
+            <button mat-stroked-button disableRipple color="accent" class="flex items-center">
+                <mat-icon class="mr-2"> check_circle </mat-icon>
+                <span>Submit</span>
+            </button>
+        </mat-card-actions>
+    </mat-card>
+</div>
```

Comment:
This codes should be wrapped in another component and then we need to call it here in the contact component as its parent